dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: Conditional Ad Units - <font color="red">NEW!</font>
+description: Using labels for conditional bids
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Conditional Bids and AdUnits
+
+Prebid 1.0 introduced the `sizeConfig` feature primarily for [responsive ad designs]({{site.baseurl}}/dev-docs/feature-responsive.html), focusing on constraining sizes based on device.
+But a number of other scenarios are supported as well:
+
+1. What if some bidders should be skipped for some devices?
+1. What if some bidders have different parameters for different devices?
+1. What if some AdUnit auctions should be skipped entirely for some devices?
```

Comment:
When I see AdUnit I think the text may be referring to a type captured in the code with specific properties, e.g., in developer docs.  If we're referring to an ad unit in prose, I think we should prefer "ad unit".